docs/framework/tutorial/shard.md

```diff
@@ -0,0 +1,353 @@
+---
+short-description: Refining and packaging the downloaded content
+...
+# Creating a shard #
+
+In this section, we'll view the blog posts we've ingested in the previous section, and make some improvements to our ingester.
+We'll add some customizations to the visual theme, and make sure videos are downloaded for offline viewing too.
+Then we will package the hatch for distribution, in an archive format called a "shard".
+
+We'll use Hatch Previewer to view the blog posts.
+If you haven't installed it yet, check how to do so in the [Install](tutorial/install.md#installing-hatch-previwer) section.
+
+## Viewing the blog posts ##
+
+Running the ingester in the last section should have created a directory with a name like `hatch_cc-blog_20180215_104946` (where the numbers are the date and time that the ingester was run.)
+Start Hatch Previewer, either by clicking its icon on the desktop, or from the command line:
+```bash
+flatpak run com.endlessm.HatchPreviewer
+```
+You'll be shown a dialog box where you select a directory.
+Navigate to the hatch directory and select it.
+
+> **NOTE:** You can also pass the hatch directory on the command line, but you have to give an absolute path due to a limitation in Flatpak.
+
+You should see a mostly blank window with a list of the ingested blog posts on the left.
+Click on one, and you should see something like this:
+
+![Screenshot of Hatch Previewer](images/hatch-previewer-screenshot.png)
+
+You can see the page as it looks when rendered in the article format, and press the "Flip" button to view the raw HTML.
+You can also see the metadata on the right-hand side.
+
+Browse through the posts, and take note of anything that looks odd.
+You can also compare them to the posts on their original website in your browser, to make sure nothing is missing.
+
+The first thing you might notice is that there are some blank spaces in the article format we used.
+These will display metadata, but haven't been filled in yet.
+There's a space after the dash at the top right, for the date that the post was published.
+This isn't pre-filled because we want to display it in whatever date format the user has selected in their settings, and that's impossible to know at this point.
```

Comment:
I know that this is a good rationale, but in this case it feels a bit like a poor excuse. We could just as well show it in the date format for the locale Hatch Previewer is running on (but not sure how easy that would be)

Reply:
Yeah, we could inject a JS script into Hatch Previewer similar to the one we inject from eos-knowledge-lib. (I'll open an issue for that.)

Reply:
https://phabricator.endlessm.com/T21646